docker/cassandra-stress/README.md

```diff
@@ -12,10 +12,10 @@ echo "${CS_DOCKER_IMAGE}" > image
 ### Building from source without shard aware driver (slower)
 
 ```
-export BRANCH=branch-5.0
+export BRANCH=use_rack_aware
+export GIT_FORK=https://github.com/sylwiaszunejko/scylla-tools-java
```

Comment:
is there some PR that we wait for? Worth to mention when we can switch back to scylla's repo

Reply:
it's example of usage from fork.

the common case is running from scylla image, and then it's from the formal repos

those build are mostly for testing, or when specific job would need a specific version (i.e. like perf tests)